### ISSUE TITLE
[DockerComposeV0] custom command does not use arguments

### ISSUE THREAD

**Issue by josefschabasser** [microsoft/azure-pipelines-tasks#13639]
## Required Information

**Question, Bug, or Feature?**  
*Type*: Bug
**Enter Task Name**: DockerComposeV0

## Environment
- Server - Azure Pipelines or TFS on-premises? Azure Pipelines
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: Corporate account
- Agent - Hosted or Private: 
    - If using Hosted agent, provide agent queue name: ubuntu-latest

## Issue Description

DockerComposeV0 does not make use of provided arguments. If one wants to pull images for specific services, this is possible by running
```
docker-compose -f myfile pull service-1 service-2
```
This task however ignores all arguments and always pulls all services:
```
/usr/local/bin/docker-compose -f /home/vsts/work/1/s/docker-infrastructure.yml -f /home/vsts/work/1/s/docker-myapp.image.yml -f /home/vsts/agents/2.175.2/.docker-compose.1601478889493.yml -p comapny/project pull
```
The tasks `task.json` hides the arguments box when using only 3 specific commands, see here: https://github.com/microsoft/azure-pipelines-tasks/blob/11d061afd2cf75ebc96bace479b19afb0a1dfc75/Tasks/DockerComposeV0/task.json#L274
The `Build service images` command utilizes the arguments: see here: https://github.com/microsoft/azure-pipelines-tasks/blob/11d061afd2cf75ebc96bace479b19afb0a1dfc75/Tasks/DockerComposeV0/dockercomposebuild.ts#L68
But the command `Run a Docker Compose command` does not respect them, see here: https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/DockerComposeV0/dockercomposecommand.ts

Please add arguments to custom commands. Thanks :-)

### Task logs

[Enable debug logging and please provide the zip file containing all the logs for a speedy resolution]

## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

### Error logs

[Insert error from the logs here for a quick overview]

**Comment by JamesMiera** [comment]
I'm curious if anyone is looking into this?

**Comment by ds-ms** [comment]
@Ganeshrockz could you take a look

**Comment by ds-ms** [comment]
@JamesMiera / @josefschabasser I apologise for the delay.  This seems like a UI issue, for which as a work around, you could basically just change it to a different command which makes the input field visible, add your arguments and then switch back to your custom command after. 

You should still be able to use `arguments` for custom commands in your YAML pipelines.

**Comment by josefschabasser** [comment]
Funny this got implemented after someone wrote "it's an UI issue"...

**Comment by ds-ms** [comment]
>Funny this got implemented after someone wrote "it's an UI issue" and afterwards deleted his/her comment... 

Haha, it was my bad, I got the issue wrong. @Ganeshrockz and I synced up offline to discuss the fix.

**Comment by GrumpyMeow** [comment]
I also ran into this issue:
```
   dockerComposeCommand: 'pull'
   arguments: '--quiet --include-deps mymainservicename
```

**Comment by josefschabasser** [comment]
> I also ran into this issue:
> 
> ```
>    action: 'Run a Docker Compose command'
>    dockerComposeCommand: 'pull'
>    arguments: '--quiet --include-deps mymainservicename
> ```
> 
> Temporary workaround:
> 
> ```
>    action: 'Run a Docker Compose command'
>    dockerComposeCommand: 'pull --quiet --include-deps mymainservicename'
>    #https://github.com/microsoft/azure-pipelines-tasks/issues/13639
>    #arguments: '--quiet --include-deps mymainservicename
> ```

Are you using an old version of the DockerCompose task?
The PR was merged, tasks v0.179.0 and newer contain the fix.

**Comment by GrumpyMeow** [comment]
We are using Azure DevOps, so I don’t think we have control over which version of a task we use. 
I’ll look into the “missing quote”, maybe that’s the root cause. But I would expect to see an error to occur. Thanks
